spec/statesman/adapters/active_record_spec.rb

```diff
@@ -343,12 +341,35 @@
         end
       end
     end
+
```

Comment:
do we want to add in one of the current tests a check that it is connected to the primary?